browser/extensions/BUILD.gn

```diff
@@ -63,6 +64,13 @@ source_set("extensions") {
     ]
   }
 
+  if (moonpay_enabled) {
+    sources += [
```

Comment:
I think you're missing some deps here like `//brave/components/moonpay/*`

Reply:
added